lib/wasm-binding-core.ts

```diff
@@ -25,8 +25,8 @@ export interface WasmCallArgumentTypeMap {
   int32: number;
   float32: number;
   float64: number;
-  int32ptr: ReadonlyArray<number>|Int32Array|null;
-  float32ptr: ReadonlyArray<number>|Float32Array|null;
+  int32ptr: ReadonlyArray<number>|Uint32Array|Int32Array|null;
+  float32ptr: ReadonlyArray<number>|Int32Array|Uint32Array|Float32Array|null;
```

Comment:
I think for WASM, all basic types (`int32`, `uint32`, `float`, ...) need to be treated differently (not like javascript there is just a `number`). When used in the context of a specific function, for example, `_add_f32`, an integer type should not work.